spec/controllers/api/base_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+describe Api::BaseController, type: :controller do
```

Comment:
Same question as earlier. Can we use this response to solve for bad requests in the client? 
[Client/Issue 9](https://github.com/midohiofoodbank/freshtrak-client/issues/9)